app/build.gradle

```diff
@@ -0,0 +1,69 @@
+plugins {
+    id 'com.android.application'
```

Comment:
This should not be an application. This needs to be configured as a library. 
Refer to `https://github.com/Codigami/CFAlertDialog/blob/master/cfalertdialog/build.gradle`